### PR TITLE
ENG-111: Implement billing usage

### PR DESCRIPTION
**Pull request by afolasope**

### CODE REVIEW

File: apps/frontend/src/modules/workspace/utils/index.ts [hunk 1]
```diff
@@ -37,6 +43,7 @@ export const planDetails = [
         current: 250,
```
Comment (jdamilola) [inline apps/frontend/src/modules/workspace/utils/index.ts:43]: @afolasope Why is this still static?

File: apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx [hunk 2]
```diff
@@ -24,36 +48,70 @@ export const Billing = () => {
                     </div>
                     <div className="space-y-6">
                         {planDetails.map(({ icon: Icon, ...plan }) => {
+                            const count =
```
Comment (jdamilola) [inline apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx:51]: @afolasope Can you simplify this conditional statement? It's very difficult to. read

File: apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx [hunk 3]
```diff
@@ -24,36 +48,70 @@ export const Billing = () => {
...
+                                plan.category === BillingCategoriesEnum.EVENTS
+                                    ? parsedBilling.events.count
+                                    : plan.category === BillingCategoriesEnum.PERSONS
+                                      ? parsedBilling.persons.count
+                                      : plan.category === BillingCategoriesEnum.RESPONSES
+                                        ? parsedBilling.responses.count
+                                        : 0;
+
+                            const limit =
```
Comment (jdamilola) [inline apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx:60]: @afolasope This too

**Comment by jdamilola** [comment]
@afolasope Can you resolve this conflict here also?

File: apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx [hunk 4]
```diff
@@ -1,59 +1,109 @@
...
+    };
+
     return (
         <div className="w-[675px] pt-10 text-intg-text-4">
             <div className="w-[515px]">
                 <h3 className="font-semibold">Manage Billing</h3>
                 <p className="text-sm">
-                    Manage your billing information and invoice. for question about billing Contact
-                    Integraflow@gmail.com or Github
+                    Manage your billing information and invoices. For questions about billing, contact
```
Comment (jdamilola) [inline apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx:45]: Can you remove the `For questions about billing, contact...`.

File: apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx [hunk 5]
```diff
@@ -1,59 +1,109 @@
...
+    events: BillingValues;
+    persons: BillingValues;
+    responses: BillingValues;
+};
+
 export const Billing = () => {
+    const { organizations } = useAuth();
+    const { workspace } = useWorkspace();
+
+    const currentWorkspace = organizations.find((org) => {
```
Comment (jdamilola) [inline apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx:21]: Why do you need to do this? Is `workspace` from `useWorkspace` not enough to get `billingUsage`?

**Review by jdamilola — CHANGES_REQUESTED** [review]

**Review by jdamilola — DISMISSED** [review]

File: apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx (hunk 4 above)
Comment (jdamilola) [reply apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx:39]: @afolasope Please don't forget to update this

Comment (afolasope) [reply apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx]: > @afolasope Please don't forget to update this

Done

**Review by jdamilola — APPROVED** [review]

**Pull request merged**